includes/class-admin.php

```diff
@@ -191,4 +191,16 @@ public static function admin_init() {
 	 */
 	public static function enqueue_scripts() {
 	}
+
+	/**
+	 * Has experimental auditing features?
```

Comment:
nit: A return tag is missing and would be helpful here and in a handful of other methods in this PR

Reply:
Done in 121cc686236bac0458e5a8e811fa2380fca29171